code/_onclick/other_mobs.dm

```diff
@@ -82,7 +82,7 @@
 		return
 	setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
 	if(istype(A,/mob/living))
-		if(!istype(natural_weapon) || a_intent == I_HELP)
+		if(!get_natural_weapon() || a_intent == I_HELP)
```

Comment:
Maybe move this after the intent comparison? Tiny microoptimization.

Reply:
sure why not